src/Tests/dotnet-new.Tests/CommonTemplatesTests.cs

```diff
@@ -887,10 +601,9 @@ public class Class1
         [InlineData("TargetFramework", "net7.0", "Class Library", "classlib", "VB", null)]
         [InlineData("Nullable", null, "Class Library", "classlib", "VB", "netstandard2.0")]
 
-        public void SetPropertiesByDefault(string propertyName, string? propertyValue, string expectedTemplateName, string templateShortName, string? language, string? framework)
+        public async void SetPropertiesByDefault(string propertyName, string? propertyValue, string expectedTemplateName, string templateShortName, string? language, string? framework)
```

Comment:
imo, this test (and below) may be removed if the content is already tested in others.